generators/java/model/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.0] - 2024-05-28
+
+- Fix: Undiscriminated unions are now generated with de-conflicted method signatures. Previously, certain
+  undiscriminated unions would have failed to compile due to Java's type erasure causing conflicts.
```

Comment:
if you have entirely deconflicted undiscriminated unions (this is going to cause the SDK to break). Here is an [example](https://github.com/merge-api/merge-java-client/blob/6cc427ce66b2d8964677a110443d311e11f2027f/src/main/java/com/merge/api/resources/ats/types/OfferCreator.java#L60-L68)